zerver/models.py

```diff
@@ -3399,78 +3399,43 @@ class UserPresence(models.Model):
     """
 
     class Meta:
-        unique_together = ("user_profile", "client")
         index_together = [
-            ("realm", "timestamp"),
+            ("realm", "last_active_time"),
+            ("realm", "last_connected_time"),
         ]
 
     id: int = models.AutoField(auto_created=True, primary_key=True, verbose_name="ID")
     user_profile: UserProfile = models.ForeignKey(UserProfile, on_delete=CASCADE)
```

Comment:
Hmm, do we perhaps want this to be either `unique=True` or `OneToOneField`?

Reply:
Yes, I think `unique=True` is missing here.